testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestHiveViews.java

```diff
@@ -13,5 +13,71 @@
  */
 package io.trino.tests.hive;
 
+import io.trino.tempto.Requires;
+import io.trino.tempto.assertions.QueryAssert;
+import io.trino.tempto.fulfillment.table.hive.tpch.ImmutableTpchTablesRequirements;
+import org.testng.annotations.Test;
+
+import static io.trino.tempto.assertions.QueryAssert.Row.row;
+import static io.trino.tempto.assertions.QueryAssert.assertThat;
+import static io.trino.tempto.query.QueryExecutor.query;
+import static io.trino.tests.TestGroups.HIVE_VIEWS;
+import static io.trino.tests.utils.QueryExecutors.onHive;
+import static io.trino.tests.utils.QueryExecutors.onPresto;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@Requires({
+        ImmutableTpchTablesRequirements.ImmutableNationTable.class,
+        ImmutableTpchTablesRequirements.ImmutableOrdersTable.class,
+})
 public class TestHiveViews
-        extends AbstractTestHiveViews {}
+        extends AbstractTestHiveViews
+{
+    @Test(groups = HIVE_VIEWS)
+    public void testFailingHiveViewsForInformationSchema()
+    {
+        // The expected behavior is different across hive versions. For hive 3, the call "getTableNamesByType" is
+        // used in ThriftHiveMetastore#getAllViews. For older versions, the fallback to doGetTablesWithParameter
+        // is used, so information_schema.views does not include translated Hive views.
```

Comment:
Make sure it's clear we're talking about Trino's `information_schema`, not the one in HMS

Reply:
Please move the comment just above assertion queries.